src/components/BaseComponents/FormGroup/FormGroup.tsx

```diff
@@ -71,7 +71,7 @@ export default function FormGroup({
           {removeRedundantString(errMessage)}
         </p>
       )}
-      <div>{children}</div>
```

Comment:
I'm sure you've checked but is the "div" definitely redundant for other components?